charts/reform-scan-blob-router/values.preview.template.yaml

```diff
@@ -41,6 +41,7 @@ java:
     STORAGE_BULKSCAN_URL: https://bulkscanaat.blob.core.windows.net
 
     STORAGE_URL: "https://$(STORAGE_ACCOUNT_NAME).blob.core.windows.net"
+    STORAGE_REFORM_CONNECTION_STRING: "DefaultEndpointsProtocol=https;BlobEndpoint=https://$(STORAGE_ACCOUNT_NAME).blob.core.windows.net;AccountName=$(STORAGE_ACCOUNT_NAME);AccountKey=$(STORAGE_ACCOUNT_KEY);"
```

Comment:
do you intend to use internal or app gw url ?

Reply:
for  pr  it is internal

Reply:
aye, `values.yaml` should use app gw whereas preview is fine here

Reply:
`STORAGE_REFORM_CONNECTION_STRING` nothing in values yaml ?

Reply:
- storage-reform-connection-string